accounts/abi/bind/base.go

```diff
@@ -119,6 +118,12 @@ func (c *BoundContract) Transact(opts *TransactOpts, method string, params ...in
 	return c.transact(opts, &c.address, input)
 }
 
+// Transfer initiates a plain transaction to move funds to the contract, calling
+// its default method if one is available.
+func (c *BoundContract) Transfer(opts *TransactOpts) (*types.Transaction, error) {
+	return c.transact(opts, &c.address, nil)
```

Comment:
As you've said zero is a valid amount if you just want to call the default function, though the use of that is getting more and more limited. (See, this is why I put these in a special struct as you need a much more involved understanding as to what and how the contract works to call these opposed to the ABI defined methods).